src/main/java/com/sparta/gourmate/domain/user/service/AddressService.java

```diff
@@ -0,0 +1,68 @@
+package com.sparta.gourmate.domain.user.service;
+
+import com.sparta.gourmate.domain.user.dto.AddressRequestDto;
+import com.sparta.gourmate.domain.user.dto.AddressResponseDto;
+import com.sparta.gourmate.domain.user.entity.Address;
+import com.sparta.gourmate.domain.user.entity.User;
+import com.sparta.gourmate.domain.user.repository.AddressRepository;
+import com.sparta.gourmate.domain.user.repository.UserRepository;
+import com.sparta.gourmate.global.exception.AddressNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@Service
+public class AddressService {
+
+    private final AddressRepository addressRepository;
+    private final UserRepository userRepository;
+
+    // 주소 등록
+    public AddressResponseDto createAddress(Long userId, AddressRequestDto requestDto) {
+        Optional<User> userOptional = userRepository.findById(userId);
+        if (userOptional.isEmpty()) {
+            throw new IllegalArgumentException("User not found");
```

Comment:
```suggestion
            throw new CustomException(ErrorCode.에러코드에 작성한 코드);
```

예외 처리를 CustomException 로 하면 예외 처리를 각 상황에 맞춰서 예외를 정의할 수 있고 더 구체적으로 예외를 처리할 수 있어요!